example_config/EUR-Lex-57k/bigru_lwan.yml

```diff
@@ -20,12 +20,11 @@ monitor_metrics: ['Micro-F1', 'Macro-F1', 'Another-Macro-F1', 'P@1', 'P@5', 'RP@
 val_metric: P@1
 
 # model
-model_name: BiGRU
-init_weight: null
+model_name: BiGRULWAN
```

Comment:
Should  `example_config/MIMIC-50/bigru.yml` be updated?
https://github.com/ASUS-AICS/LibMultiLabel/blob/master/example_config/MIMIC-50/bigru.yml

Reply:
Updated, thanks.